### PR TITLE
#86 - add prometheus gauge to count how many panics are caught within…

### PR DESCRIPTION
**Pull request by gabrielruiu**
… http request handlers

### CODE REVIEW

File: http/metrics.go [hunk 1]
```diff
@@ -52,11 +52,16 @@ var (
 		},
 		[]string{"code", "method", "source"},
 	)
+
+	paceHTTPPanicCounter = prometheus.NewGauge(prometheus.GaugeOpts{
```
Comment (threez) [inline http/metrics.go]: could you add the new metric to the readme?

File: http/router.go [hunk 2]
```diff
@@ -23,7 +23,7 @@ func Router() *mux.Router {
 	r.Use(metricsMiddleware)
 
 	// last resort error handler
-	r.Use(errors.Handler())
+	r.Use(errors.HandlerWithCounter(paceHTTPPanicCounter))
```
Comment (threez) [inline http/router.go]: Can we maintain the `errors.Handler` interface for backwards compatibility. I don't think we need both forms.

Comment (gabrielruiu) [reply http/router.go]: Added a modification such that the `errors.Handler()`  stays the same

File: http/metrics.go (hunk 1 above)
Comment (gabrielruiu) [reply http/metrics.go]: added reference to the metric in the `metrics/README.md`, but it feels kinds of out-of-place. If there is a better place to put it, let me know

**Pull request merged**